src/BenchmarksApps/Kestrel/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.0.30" />
+    <PackageReference Include="System.Text.Json" Version="6.0.0-preview.7.21316.4" />
```

Comment:
We still need to decide on the strategy to run with these changes.
Issue here is that it's pinning the STJ's version so we'll potentially miss some regressions. Also it's using non-crossgened versions which will have an impact on startup (not sure how much) and trigger some regressions.
Finally we are now running net5.0 baseline with STJ from net6.0.

Right now the best solution would be to just add some ifdefs for net6.0 and use source generators in this case.

Reply:
Thanks. I've made the change to ifdef the implementation using source generation to .NET 6 and higher only. To resolve the issue with non-crossgen'ing & using a pinned version of STJ - we've settled on a [proposal](https://github.com/dotnet/designs/blob/02486c962c09113deec777cabd9a54e9acfef437/accepted/2021/InboxSourceGenerators.md) for having inbox source generators. It should be implemented soon & at that time we can have STJ version with the rest of the framework. cc @ericstj 

Are we currently running net6.0 benchmarks continuously (CI/dashboards etc)? I notice the only TFM in this project is net5.0.

Reply:
With 6.0 Preview7 builds this is now the case.  No `PackageReference` is needed to use the source generator.

Reply:
Awesome, I've removed the reference. I saw that the internal .NET 6 benchmark runs are using the latest Preview 7 bits, so I believe this PR is unblocked. @davidfowl, can we merge now? (Sebastien is currently OOF)

Reply:
@halter73 It seems that both @sebastienros and @davidfowl are offline, could you please take a look (and hopefully merge)?